common/menu.c

```diff
@@ -27,5 +27,36 @@ int read_menu()
             printf("Wrong algorithm choice, try again?\n");
         }
     } while (choice < 0 || choice > TOTAL_ALGOS);
+    if(choice==8)
+        {
+        printf("you selected shell sort")
+            int data[] = {9, 8, 3, 7, 5, 6, 4, 1};
```

Comment:
Data should not be pre-defined, but should be take from common imput. Also use single PR for all Shell Sort related commits.

Reply:
Take reference from other sorting algos to use the main.c